scripts/load_testing/generate.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+conn_configs=(100 1000 10000)
+mem_configs=(4 8 16 32)
+size_configs=(64 128 256 512 1024 2048)
+instances=30
+ksize=32
+rate=100000
+threads=2
```

Comment:
Remember that threads is a multiplier on connections. The `--connections` param for rpc-perf is actually the per-endpoint poolsize for each client thread. I think in the rpc-perf configs section you'd want to take the ceiling of the connections divided by the rpc-perf threads

Reply:
Yep, that's what I'm doing right now: https://github.com/twitter/pelikan/blob/9f791895c494a51eb5c252587c21dc00af72923d/scripts/load_testing/client_config.py#L27 and https://github.com/twitter/pelikan/blob/9f791895c494a51eb5c252587c21dc00af72923d/scripts/load_testing/client_config.py#L62